src/nbl/CMakeLists.txt

```diff
@@ -464,7 +464,7 @@ if (NBL_COMPILE_WITH_CUDA)
 endif()
 
 target_compile_definitions(Nabla 
-	PRIVATE	_NBL_STATIC_LIB_ SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS $<$<BOOL:${WIN32}>:VK_USE_PLATFORM_WIN32_KHR>
+	PRIVATE	_NBL_STATIC_LIB_ EGL_STATIC SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS $<$<BOOL:${WIN32}>:VK_USE_PLATFORM_WIN32_KHR>
```

Comment:
does EGL then statically link EGL and X11 on Linux?

Reply:
its kinda important that EGL (our spoof EGL) doesn't rely on weird shit like `-lEGL` or `-lX11` on Linux, and does a `dlopen` to get them (allowing it to not crash and burn on systems without Wayland or X11)

Reply:
eveyrthing is loaded dynamically